Gemfile

```diff
@@ -9,8 +9,8 @@ end
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.1.4'
-# Use postgresql as the database for Active Record
-gem 'pg', '~> 0.18'
```

Comment:
Shouldn't we be requiring both `pg` and `mysql2` in the interim period, while we're transitioning from Heroku + PostgreSQL to Wikimedia Toolforge + MySQL?

Reply:
Oops yes, good catch! Have put that back in 61cd88b78cca943fd5d88c4052e9dc2d58cd7eac.